www/app/Http/Middleware/EnsureSetupComplete.php

```
@@ -18,14 +20,77 @@ public function __construct(
+        if ($this->settings->isAuthBypassPermanent()) {
+            return $next($request);
+        }
+
+        // Check if authentication is bypassed (temporarily via session)
+        if ($request->session()->get('auth_bypass_session')) {
             return $next($request);
         }
 
-        // If setup is not complete, redirect to setup wizard
+        // If no users exist, redirect to setup wizard (first-run)
+        if (User::count() === 0) {
```

PHPStan (2.1.46)

[error] Call to an undefined static method App\Models\User::count().


(staticMethod.notFound)